stores/sql/types.go

```diff
@@ -36,11 +36,31 @@ type (
 )
 
 var (
-	_ sql.Scanner = &Currency{}
-	_ sql.Scanner = &FileContractID{}
-	_ sql.Scanner = &Hash256{}
-	_ sql.Scanner = &PublicKey{}
-	_ sql.Scanner = &SecretKey{}
+	_ driver.Valuer = (*AutopilotConfig)(nil)
```

Comment:
`nit` you could've defined an interface combining `driver.Valuer` and `sql.Scanner`

Reply:
edit: probably not even a `nit` this one